cmd/migrate.go

```diff
@@ -240,7 +240,7 @@ func (ms *migrationService) status() {
 
 func (ms *migrationService) parseSubcommand(args []string) {
 	flags := flag.NewFlagSet("migrate", flag.ExitOnError)
-	flags.StringVar(&ms.DSNS, "db", "root@localhost:26257", "CockroachDB JDBC connection details.")
+	flags.StringVar(&ms.DSNS, "database.address", "root@localhost:26257", "Address of CockroachDB server (username:password@address:port/dbname)")
```

Comment:
If we're renaming the input, let's also change the `ms.DSNS` field name to stay consistent.